pkg/kubelet/cpumanager/policy_static.go

```diff
@@ -210,6 +219,11 @@ func takeByTopology(topo *topology.CPUTopology, availableCPUs cpuset.CPUSet, num
 func (p *staticPolicy) releaseCPUs(s state.State, release cpuset.CPUSet) {
 	// mutate the shared pool, adding supplied cpus
 	s.SetDefaultCPUSet(s.GetDefaultCPUSet().Union(release))
+	if p.underPressure {
```

Comment:
When making this change, I realized that I would like to keep the logging for `underPressure` transitions.  So the `if` block are doing to be there anyway.

I switched to using `IsEmpty()` though and do verify it is false before setting `underPressure` to `false`